server/src/constants.ts

```diff
@@ -27,6 +27,8 @@ export const WEB_ROOT = process.env.IMMICH_WEB_ROOT || '/usr/src/app/www';
 const HOST_SERVER_PORT = process.env.IMMICH_PORT || '2283';
 export const DEFAULT_EXTERNAL_DOMAIN = 'http://localhost:' + HOST_SERVER_PORT;
 
+export const PROCESS_INVALID_IMAGES = process.env.IMMICH_PROCESS_INVALID_IMAGES === 'true';
```

Comment:
Would this need to be documented as well (independent from where the option will be in the end)?

Reply:
Yes, it should be added [here](https://github.com/immich-app/immich/blob/main/docs/docs/install/environment-variables.md).

Reply:
Done